packages/transport/src/types/messages.ts

```diff
@@ -1538,6 +1548,11 @@ export type Features = {
     busy?: boolean;
     homescreen_format?: HomescreenFormat;
     hide_passphrase_from_host?: boolean;
+    internal_model?: string;
+    unit_color?: number;
+    unit_btconly?: boolean;
+    homescreen_width?: number;
+    homescreen_height?: number;
```

Comment:
We can probably use this somehow to remove `deviceModelInformation` definition in `homescreen.ts`

Not in this PR ofc